### DISCUSSION TITLE
Large-v2 Model Availability

### DISCUSSION THREAD

**Discussion by KSRider87** [Dadangdut33/Speech-Translate#24] [Ideas]
Hi there,

Been playing around with your tool, really appreciate how easy it is to run versus troubleshooting the main 
Whisper Repo, I was curious to know if it was possible to add the Whisper-V2 model for use in your next release?

I would have loved to make the addition to the code for you, but my coding skills are non existent. I have tried to cheat
the system with replacing the model names for the v2 model, but your system checks the hashes before using the model haha.
 
Thanks for your time and all the effort you put into this easy to use package!

**Comment by Dadangdut33** [comment]
thank you for your suggestion too, and yes it will be fixed/added as mentioned in #22
